samples/aspnetcore/mvc/produces/src/ProducesMatcherPolicy/ProducesMatcherPolicy.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.2</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.0-preview3-35497" />
```

Comment:
Would this feature be available in AspNetCore 2.1

Reply:
Requires 2.2 and above.

Reply:
Is there a way to get this running or backported to 2.1? My execution environment only supports 2.1 (AWS Lambda).

The [Consumes] Attribute also works in 2.1.

Reply:
No, supporting this in 2.1 would require a completely different implementation.

The final version of 2.2 has not been published. I'm sure cloud vendors will support 2.2 sometime soon after it has been released.

Reply:
As far as I know, AWS only supports .NET Core LTS Versions.

Or can a .NET Core 2.1 Runtime execute AspNetCore 2.2?

Reply:
If you need this in 2.1 - I would suggest starting with an implementation of `IActionConstraint`. This would be really similar to https://github.com/aspnet/Mvc/blob/master/src/Microsoft.AspNetCore.Mvc.Core/ConsumesAttribute.cs#L110 - but based on `Accept` instead of `Content-Type`.

Reply:
The reason this is based on 2.2 is that we have a more powerful feature set for doing this kind of thing in 2.2 - it's the evolution of `IActionConstraint`.

Reply:
Thanks a lot for the code suggestion. I wa looking for this feture for a day or two.